HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -5076,7 +5096,7 @@ def self.calculate_heat_pump_defrost_load_power_watts(heat_pump, unit_multiplier
     end
     # cooling capacity and airflow are already with unit multiplier, calculate the capacity w/o multiplier
     nominal_cooling_capacity = heat_pump.cooling_capacity / unit_multiplier
-    defrost_power_fraction = (max_heating_airflow / design_airflow)**3
+    defrost_power_fraction = calculate_fan_power_from_curve(1.0, max_heating_airflow / design_airflow, heat_pump)
```

Comment:
This should call the calculate_fan_power_from_curve in the first place, so I captured it here.